projects/addon-doc/src/components/documentation/documentation.component.ts

```diff
@@ -59,6 +61,8 @@ export class TuiDocDocumentationComponent implements AfterContentInit {
         @Inject(ChangeDetectorRef) private readonly changeDetectorRef: ChangeDetectorRef,
         @Inject(TUI_DOC_DOCUMENTATION_TEXTS)
         readonly texts: [string, string, string, string, string],
+        @Inject(TUI_DOC_EXCLUDED_PROPERTIES)
+        readonly excludedProperties: readonly string[],
```

Comment:
why array instead of set?

Reply:
as we discussed before, `set` has no performance advantage over `array` https://jsbench.me/6zlbp925nn/1
but ok